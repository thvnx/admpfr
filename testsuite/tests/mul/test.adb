with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M, N, O : Mpfloat;
begin
   N.Set (Float (1.5));
   O.Set (Float (4.0));

   M.Mul (N, O);
   Put_Line (M'Image);

   M.Mul (N, 5);
   Put_Line (M'Image);

   M.Mul (N, 6.0);
   Put_Line (M'Image);

   M.Mul_2 (N, 8);
   Put_Line (M'Image);
end Test;
