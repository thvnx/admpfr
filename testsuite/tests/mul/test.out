6.0000000000000000e+00
7.5000000000000000e+00
9.0000000000000000e+00
3.8400000000000000e+02
